### PR TITLE
Add documentation for migrated csi volume support

### PR DESCRIPTION
**Pull request by liuy1vmware**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/750/

File: docs/csi-migrated-volume-support.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# CSI migrated volume support
+
+Velero plugin for vsphere v1.4.0 and above supports backing up of migrated vsphere csi volumes. A migrated vsphere csi volume is original provisioned by vmware vsphere cloud provider but migrated to vsphere csi driver.
```
Comment (xing-yang) [inline docs/csi-migrated-volume-support.md:3]: Velero Plugin for vSphere

General comment: vsphere -> vSphere
vmware -> VMware
csi -> CSI

**Review by xing-yang — COMMENTED** [review]
Can you add a link in this page to this doc?  https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/main/docs/vanilla.md

File: docs/csi-migrated-volume-support.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# CSI migrated volume support
+
+Velero Plugin for vSphere v1.4.0 and above supports backing up of migrated vSphere vSphere volumes. A migrated vSphere CSI volume is original provisioned by VMware vSphere Cloud Provider but migrated to vSphere CSI driver.
```
Comment (xing-yang) [inline docs/csi-migrated-volume-support.md:3]: duplicate "vSphere"

original -> originally

**Comment by xing-yang** [comment]
The DCO error is still there.

File: docs/vanilla.md [hunk 3]
```diff
@@ -330,6 +330,8 @@ Upload CRD has a number of phases for the `.status.phase` field:
 
 UploadError uploads will be periodically retried.  At that point their phase will return to InProgress.  After an upload has been
 successfully completed, its record will remain for a period of time and eventually be removed.
+#### If the CNS block volume is migrated
+Refer to [CSI migrated volume support](csi-migrated-volume-support.md)
```
[the remark this answers is not in the document]
Comment (liuy1vmware) [reply docs/vanilla.md]: fixed

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/751/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/752/

File: docs/csi-migrated-volume-support.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+metadata:
+   name: velero-vsphere-plugin-feature-states
+
+kubectl -n velero apply -f velero-vsphere-plugin-feature-states.yaml
+```
+## Prerequisite
+We need to make sure vSphere CSI migration is properly enabled.
+[Migrating In-Tree vSphere Volumes to vSphere Container Storage Plug-in](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-968D421F-D464-4E22-8127-6CB9FF54423F.html)
+
+Below cmd will print all CRs, 1 for each migrated vsphere csi volume
```
Comment (xing-yang) [inline docs/csi-migrated-volume-support.md:45]: vsphere csi -> vSphere CSI

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/753/

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/754/

**Comment by xing-yang** [comment]
LGTM

**Review by xing-yang — APPROVED** [review]

**Pull request merged**